libpod/runtime_ctr.go

```diff
@@ -579,6 +579,17 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force bool,
 
 	c.newContainerEvent(events.Remove)
 
+	for _, v := range c.config.ImageVolumes {
+		// Unmount the specified image.
+		img, err := c.runtime.ImageRuntime().NewFromLocal(v.Source)
+		if err != nil {
+			return errors.Wrapf(err, "error unmounting image volume %q:%q", v.Source, v.Dest)
+		}
+		if err := img.Unmount(false); err != nil {
```

Comment:
Does this properly handle the case where the image is already unmounted? We need to be certain that case does not cause us to be unable to remove containers.

Reply:
Thanks for checking. Yes, that case is covered. If an image is already umounted, we don't get an error.